psalm.xml

```diff
@@ -248,5 +248,10 @@
                 <file name="lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php"/>
             </errorLevel>
         </ArgumentTypeCoercion>
+        <MoreSpecificImplementedParamType>
+            <errorLevel type="suppress">
+                <file name="lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php"/>
+            </errorLevel>
+        </MoreSpecificImplementedParamType>
```

Comment:
Hey by the way, why do we have this ignore in the first place?

Reply:
I guess because not all versions of doctrine/persistence (where the interface is defined) have precise docblocks.

Reply:
It’s about the doLoadMetadata method

Reply:
:thinking: but we are only running Psalm with the latest version of `doctrine/persistence`, right?

Reply:
Okay let me do a release.

Reply:
I see what you did there.